convertCsvToJson.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+const csvtojson = require('csvtojson');
+
+const csvFilePath = path.join(__dirname, 'csvdirectory', 'nodejs-hw1-ex1.csv');
+const txtFilePath = path.join(__dirname, 'output.txt');
+
+const writeStream = fs.createWriteStream(txtFilePath, { flags: 'w' });
+
+writeStream.on('error', (err) => {
+  console.error('Error writing:', err);
+});
+
+writeStream.on('finish', () => {
+  console.log('CSV data converted and written to output.txt');
+});
+
+csvtojson()
+  .fromFile(csvFilePath)
+  .subscribe((jsonObj) => {
+    const formattedJson = `{"book":"${jsonObj.Book}","author":"${jsonObj.Author}","price":${parseFloat(jsonObj.Price)}}\n`;
+
+    writeStream.write(formattedJson);
+  }, (err) => {
+    console.error('Error converting CSV to JSON:', err);
+  }, () => {
+    writeStream.end();
+  });
```

Comment:
Try to rewrite this using the `pipeline(...)` function. You should have the read stream, transform it with csvtojson, and save the final result through the write stream. Do not worry about capitalized field names in the json file. Handle errors in `pipeline(...)` only.